### PR TITLE
chore: update command generator for versioned docs

### PR DESCRIPTION
**Pull request by TerryHowe**
Created a mapping file in case we need to use a bug fix release to generate the docs. I don't think that will happen often.

### CODE REVIEW

File: versions.json
Comment (FeynmanZhou) [inline versions.json:2]: If we specify the patch version number here, it will require us update the version number per every patch release. However, we probably not need to do this since 1.2.1 and 1.2.0 docs shouldn't differ according to SemVer because there are no new features in a new patch release. 

I would suggest keeping it as minor version or use `1.1.x`, `1.2.x`.

**Review by FeynmanZhou — COMMENTED** [review]
Please see my comments above

**Comment by TerryHowe** [comment]
@FeynmanZhou I removed the bug fix release from the script and added a way to map using bug fixes if necessary. Normally, the pages won't change during a release.

File: versions.json
Comment (TerryHowe) [reply versions.json]: Done

**Review by qweeah — APPROVED** [review]
~~LGTM although IANAM.~~ What if we are updating 1.1 and the content of version-latest is 2.1.1?

Better add some comments or documents on how `versions-latest` should be used, otherwise when someone need to refresh for a patch release, they need to read and understand below code in `refresh-commands.sh`
```bash
map_version() {
    grep "$1" versions-latest || echo "$1.0"
}
```

File: tools/refresh-commands.sh
Comment (qweeah) [inline tools/refresh-commands.sh:37]: What if we are updating 1.1 and the content of version-latest is `2.1.1`?

**Comment by qweeah** [comment]
LGTM although IANAM.

File: tools/refresh-commands.sh
Comment (FeynmanZhou) [inline tools/refresh-commands.sh:18]: @TerryHowe It seems the generator script can be tested by maintainers after this PRs merged into `main`, right?

Comment (TerryHowe) [reply tools/refresh-commands.sh:18]: Yes, it will run with the `npm build`, but right now, the `npm build` does not fail if this fails.

Comment (TerryHowe) [reply tools/refresh-commands.sh:18]: As soon as this merges, I'll create a PR with the latest updates.

**Review by FeynmanZhou — APPROVED** [review]
LGTM. Thanks @TerryHowe

**Pull request merged**
